app/controllers/hearing-requirements/hearing-requirements-summary-sections.ts

```diff
@@ -630,3 +614,126 @@ function getInterpreterLanguageAnswer(language: InterpreterLanguageRefData): str
   }
   return result;
 }
+
+function buildAppellantInterpretersummaryList(interpreterRows: SummaryRow[], visibleChangeLink: boolean, hearingRequirements: HearingRequirements) {
+  interpreterRows.push(
+    getSummaryRow(visibleChangeLink,
+      i18n.common.cya.questionRowTitle,
+      [i18n.pages.hearingRequirements.accessNeedsSection.interpreterTypePage.title]
+    )
+  );
+
+  let appellantInterpreterLanguageCategory: string = '';
+  let appellantInterpreterLanguageCategoryList = hearingRequirements.appellantInterpreterLanguageCategory || [];
+  if (appellantInterpreterLanguageCategoryList && appellantInterpreterLanguageCategoryList.length > 0) {
+    appellantInterpreterLanguageCategory += appellantInterpreterLanguageCategoryList.includes('spokenLanguageInterpreter') ? 'Spoken language interpreter' : '';
+    appellantInterpreterLanguageCategory += (appellantInterpreterLanguageCategoryList.length === 2) ? Delimiter.BREAK_LINE : '';
```

Comment:
Not sure what this magic number 2 is. Why not define it for clarity and better understanding.